youtube_dl/extractor/npo.py

```diff
@@ -184,21 +186,29 @@ def _real_extract(self, url):
         return self._get_info(url, video_id) or self._get_old_info(video_id)
```

Comment:
The site used in `_get_old_info()` no longer exists. Make this change and also delete the old method.
 ```suggestion
         return self._get_info(url, video_id)
```